### PR TITLE
[WIP] Add support for Windows 2019 RTM

### PR DESCRIPTION
**Pull request by aserdean**
Signed-off-by: Alin Gabriel Serdean <aserdean@ovn.org>

### CODE REVIEW

File: contrib/roles/windows/version_check/vars/windows.yml [hunk 1]
```diff
@@ -2,3 +2,4 @@
 supported_versions_by_the_playbook:
   - "{{windows1709}}"
   - "{{windows1803}}"
+  - "{{windows2019}}"
```
Comment (lanoxx) [inline contrib/roles/windows/version_check/vars/windows.yml]: Also add 1809 please.

**Comment by lanoxx** [comment]
What is preventing this request from being merged?

**Comment by aserdean** [comment]
I need to rebase and update the docs.

I wanted to give it a last round of testing once the evaluation ISOs are up (i.e.: https://www.microsoft.com/en-us/evalcenter/evaluate-hyper-v-server-2019) and after merge this PR.

AFAIK there is no planned time to release the ISOs, do you need this PR merged sooner?

**Comment by lanoxx** [comment]
This patch does not only apply to Server 2019, but also to the semi-annual release channel, e.g. Windows Server 1809. For Windows Server 1809 the images are already available on AWS.

**Comment by aserdean** [comment]
@lanoxx the difference between Server 2019 and 1809 is mainly how updates are rolled out and for how long they will be rolled out, the rest is the same.

FYI: I'm testing with a Server 2019.

I'll rebase the PR today and give it another round of testing.

**Pull request closed without merging**